Cargo.toml

```diff
@@ -68,6 +68,9 @@ panic = "abort"
 codegen-units = 1
 lto = "fat"
 
+[patch.crates-io]
+revm = { git = "https://github.com/refcell/revm", branch = "rf/deposit-tx-fix" }
+
```

Comment:
This needs to be removed before merging.

The revm versions should be bumped once a release is cut.